app/serializers/api/v3/top_profiles_serializer.rb

```diff
@@ -15,7 +15,12 @@ def node_type
       end
 
       def photo_url
-        object&.top_profile_image&.image&.url
+        if object.top_profile_image
+          object.top_profile_image.image.url
+        else
+          top_profile = Api::V3::TopProfile.includes(:context).find(object.id)
```

Comment:
why do we need this? isn't object == top_profile?

Reply:
it is, but it's an instance and I wanted to avoid n+1 queries (i'm getting info from the `context` in the service)